src/setup/unattended-install/config-tool-ref.md

```diff
@@ -80,28 +105,28 @@ ConfigurationTool.com {/SetupInstall <platform_db_admin_username> <platform_db_a
 
 :   Prints the serial number of this installation.
 
+`/SetPlatformServerAdminPassword <platform_server_admin_password>`
+
+:   Defines the password for the Platform Server Admin user, if the user is active.  
+    Note: This command does not work when using Integrated Authentication.
+    
 `/GetDeploymentZones`
 
 :   Lists the configured deployment zones for the current installation. The data is presented in JSON format, limited to the relevant settings that can be manipulated with `/ModifyDeploymentZone`, namely: the Configuration Name and Address, and it's Enable HTTPS status.
 
-`/ModifyDeploymentZone <configuration_name> <zone_address> [<enable_https>]`
+`/ModifyDeploymentZone <deployment_zone_name> <deployment_zone_address> [<enable_https>]`
 
 :   Modifies the Address and/or Enable HTTPS settings of a Deployment Zone.
 
-    `<configuration_name>` is the Deployment Zone that you want to modify; this argument is case insensitive (for example "GLOBAL" will map to "Global").
+    `<deployment_zone_name>` is the Deployment Zone that you want to modify; this argument is case insensitive (for example "GLOBAL" will map to "Global").
 
-    `<zone_address>` is the new address for the target Deployment Zone.
+    `<deployment_zone_address>` is the new address for the target Deployment Zone.
 
     `[<enable_https>]` is an optional boolean argument; if this argument is not provided the setting will remain unchanged. If the string "true" (case insensitive) is provided, the Enable HTTPS setting is set to "true"; if any other string is provided the setting Enable HTTPS is set to "false". The applied value of the Enable HTTPS setting is displayed.
 
 `/CreateUpgradeCacheInvalidationService`
 
 :   Installs cache invalidation service or reconfigures the service given in the configuration file (`server.hsconf`).
-
-`/SetPlatformServerAdminPassword <platform_server_admin_password>`
-
-:   Defines the password for the Platform Server Admin user, if the user is active.  
-    Note: This command does not work when using Integrated Authentication.
```

Comment:
We're dropping this sentence... Should we keep it in the new location (`/UpgradeInstall`)?